main.js

```diff
@@ -162,10 +162,134 @@ const inventory = [
-        stockToSell += inventoryKey.originalStock - inventoryKey.sold;
+        stockToSell += inventory[inventoryKey].originalStock - inventory[inventoryKey].sold;
     }
     return stockToSell;
-}
\ No newline at end of file
+}
+
+function calculateRevenueTarget(inventoryArray) {
+    let revenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        revenue += (inventoryArray[inventoryKey].originalStock - inventoryArray[inventoryKey].sold) * inventoryArray[inventoryKey].price;
+    }
+    return revenue;
+}
+
+function calculateCurrentRevenue(inventoryArray) {
+    let currentRevenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        currentRevenue += inventoryArray[inventoryKey].sold * inventoryArray[inventoryKey].price;
+    }
+    return currentRevenue;
+}
+
+const revenueTarget = document.getElementById("revenueTarget");
+revenueTarget.textContent += calculateRevenueTarget(inventory);
+revenueTarget.style.color = "blue";
+
+const sellingStock = document.getElementById("stockToSell");
+sellingStock.textContent += calculateAmountToSell(inventory);
+sellingStock.style.color = "red";
+
+const currentRevenue = document.getElementById("currentRevenue");
+currentRevenue.textContent += calculateCurrentRevenue(inventory);
+currentRevenue.style.color = "green";
+
+const typeOfTVOne = document.getElementById("typeOfTVOne");
+typeOfTVOne.textContent += inventory[1].type;
+
+const typeOfTVTwo = document.getElementById("typeOfTVTwo");
+typeOfTVTwo.textContent += inventory[2].type;
+
+const revenueContainer = document.getElementById("revenueContainer");
+
+function formatInventoryArray(inventory) {
+    for (let i = 0; i < inventory.length; i++) {
+        const tvFormatted = document.createElement("p");
+        tvFormatted.setAttribute("Id", "tvFormatted" + i);
+        tvFormatted.setAttribute('style', 'white-space: pre;');
+        tvFormatted.textContent = formatTVName(inventory[i]) + " \n" +
+            formatPrice(inventory[i].price) + " \n" +
+            formatTVSize(inventory[i].availableSizes);
+        revenueContainer.appendChild(tvFormatted);
+    }
+}
+
+formatInventoryArray(inventory);
+
+function formatTVName(television) {
+    return television.brand + " " + television.type + " - " + television.name + " \r";
+}
+
+function recalculateScreenSizeToCM(sizeInInches) {
+    return (sizeInInches / 0.39370).toFixed(0);
+}
+
+function formatTVSize(televisionSize) {
+    let screenSizeString = "";
+    for (const televisionSizeKey in televisionSize) {
+        if (!(televisionSize[televisionSizeKey] === televisionSize[televisionSize.length - 1])) {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm) | ";
+        } else {
+            screenSizeString += televisionSize[televisionSizeKey] + " inches " + "(" + recalculateScreenSizeToCM(televisionSize[televisionSizeKey]) + " cm)";
```

Comment:
Als je tussen woorden iets moet plaatsen en dus niet op het eind, dan is het aan te raden te kijken naar de `string.join` functie. Voorbeeld:
```
const sizes = ["32", "43", "55"];
console.log(sizes.join(" | ");
```